app_integrations/apps/app_base.py

```diff
@@ -204,23 +204,35 @@ def _check_http_response(self, response):
 
         return success
 
-    def _make_request(self, full_url, headers, params):
-        """Method for returning the json loaded response for this request
+    def _make_get_request(self, full_url, headers, params):
```

Comment:
Let's make the `params` arg be an optional that defaults to None:

`def _make_get_request(self, full_url, headers, params=None):..`

And then update your call [here](https://github.com/airbnb/streamalert/pull/355/files#diff-02782405a17cef489b01b36d62d22196R109) so it doesn't have to pass `None` explicitly